_data/other.yml

```diff
@@ -279,6 +279,16 @@ websites:
       img: promoter.png
       tfa: No
       twitter: Promoter_io
+      
+    - name: Repairshopr
+      url: http://www.repairshopr.com/
```

Comment:
Sorry, all login pages and the actual product use *https* but everything else uses http.